application/widget/shell.py

```diff
@@ -1,8 +1,13 @@
 import os
-import pty
+
```

Comment:
Some formatting errors in this file.  

You can run `make format` to see what is causing this.  
Sometimes `make format-fix` can fix it for you (depending on the issue).